.ci/script.sh

```diff
@@ -3,7 +3,9 @@
 set -e
 set -o pipefail
 
-if [[ -n "${CI_TARGET}" ]]; then
+if [[ "${CI_TARGET}" == "lint" ]]; then
+  make lint testlint
+elif [[ -n "${CI_TARGET}" ]]; then
```

Comment:
Instead of this having this logic in `.ci/*`, can we instead have the `lint` target in the Makefile also run `testlint`? We've made some progress towards removing special cases from the CI scripts and travis.yml, and it would be good to continue that direction.

If for some reason people find it annoying for local development, we can provide an additional `clint` target in the Makefile that only runs clint.